packages/inter-protocol/src/vpool-xyk-amm/addLiquidity.js

```diff
@@ -34,8 +27,23 @@ const makeMakeAddLiquidityInvitation = (zcf, getPool) => {
     return pool.addLiquidity(seat);
   };
 
+  const AddLiquidityProposalShape = M.splitRecord({
```

Comment:
Everything in `vpool-xyk-amm` can be omitted because there's an open PR to delete. I'll skip review of these.

Reply:
@Chris-Hibbert 's deletion PR beat me to it ;)

Gone.